file.go

```diff
@@ -15,29 +15,39 @@ import (
 type readType string
 
 const (
-	readDotenv readType = "dotenv"
-	readJson   readType = "json"
-	deferred   readType = "" // defer file config type to filename suffix
+	rDotenv      readType = "dotenv"
+	rJSON        readType = "json"
+	rJSONComplex readType = "json{}" // complex json key value pair: {"k":{"v1":[],"v2":[]}}
+	rWhole       readType = "whole"  // indicates to associate the entirety of a file to the given key name
```

Comment:
`whole` is good. But given that it essentially means to not parse the file, might it better be expressed as `raw`?

Reply:
* After talking this over, I mentioned that the entire file is infact parsed
* A JSON file would be stored and used as a `yaml.Node` 
* Any read JSON file must first be read and only then convered to a `yaml.Node` obj before it’s sent/output back into JSON
https://github.com/Bestowinc/cogs/blob/1018473ac133f14c0c1edefaaf1671dd4de0a092/file.go#L155-L161